.github/workflows/release_windows.yml

```diff
@@ -88,24 +88,17 @@ jobs:
           $latestYml.files[0].size = $size
```

Comment:
the artifact path would need to be updated here as well if we change the exe name

Reply:
can we lowercase the sha512 here? it is part of the AC here https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/issues/590

Reply:
i am referring to lines 82-88 btw. github won't let me highlight them so this is as close as i can get

Reply:
SHA sums are not case-sensitive. I had this question as well:
https://superuser.com/questions/1712250/do-lower-and-capital-letters-matter-for-sha-checksum

Reply:
yes but it fails the check client side and refuses to auto update on windows if the cases are not identical. I believe this check is occurring in the `electron-updater` `autoUpdater`. Right now I have to manually lower case the hash before each release

Reply:
thats really weird though. I will investigate this further.

Reply:
ok, should be fixed now. The string will be lower case after being generated